src/debug_commads.c

```diff
@@ -361,6 +362,31 @@ DEBUG_COMMAND(GCForceBGInvoke) {
   return REDISMODULE_OK;
 }
 
+DEBUG_COMMAND(ttl) {
+  if (argc < 1) {
+    return RedisModule_WrongArity(ctx);
+  }
+  IndexLoadOptions lopts = {.flags = INDEXSPEC_LOAD_NOTIMERUPDATE,
+                            .name = {.cstring = RedisModule_StringPtrLen(argv[0], NULL)}};
```

Comment:
Do we need to strdup this in `IndexSpec_LoadEx()`?

Reply:
No its not using it after finding the spec